tools/testing/e2e/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+load("//build_tools/bazel:build_defs.oss.bzl", "iree_runtime_cc_library")
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = ["layering_check"],
+    licenses = ["notice"],  # Apache 2.0
+)
+
+iree_runtime_cc_library(
+    name = "e2e_test_util",
+    srcs = ["test_utils.cc"],
+    hdrs = ["test_.utils.h"],
```

Comment:
Typo here
```suggestion
    srcs = ["test_utils.cc"],
    hdrs = ["test_utils.h"],
```

Reply:
Thanks! I need to test the bazel build as well.